src/python_testing/TC_RVCCLEANM_2_1.py

```diff
@@ -92,16 +92,39 @@ async def test_TC_RVCCLEANM_2_1(self):
         self.is_ci = self.check_pics("PICS_SDK_CI_ONLY")
         if self.is_ci:
             app_pid = self.matter_test_config.app_pid
-            if app_pid == 0:
-                asserts.fail("The --app-pid flag must be set when PICS_SDK_CI_ONLY is set")
-            self.app_pipe = self.app_pipe + str(app_pid)
-
-        asserts.assert_true(self.check_pics("RVCCLEANM.S.A0000"), "RVCCLEANM.S.A0000 must be supported")
-        asserts.assert_true(self.check_pics("RVCCLEANM.S.A0001"), "RVCCLEANM.S.A0001 must be supported")
-        asserts.assert_true(self.check_pics("RVCCLEANM.S.C00.Rsp"), "RVCCLEANM.S.C00.Rsp must be supported")
-        asserts.assert_true(self.check_pics("RVCCLEANM.S.C01.Tx"), "RVCCLEANM.S.C01.Tx must be supported")
-
-        attributes = Clusters.RvcCleanMode.Attributes
+            if app_pid != 0:
```

Comment:
I like this. Should this be indented one less together with like 94 and line 93 be removed?

Same comment in clean 2.2 and run 2.1 and 2.2.

Reply:
I believe you are correct, I have unindented lines 95 and 94 as the self.is_ci condition check on line 93 does not appear to be needed after running some tests without it. I have done the same in clean 2.2 and run 2.1 and 2.2 test modules as well.

Reply:
Sorry, I have re-added the if self.is_ci checks as they appear to be needed for these tests